tests/test_cortex_endpoint.py

```diff
@@ -0,0 +1,36 @@
+from thehive4py.client import TheHiveApi
+
+
+class TestCortexEndpoint:
+    def test_list_analyzers(self, thehive: TheHiveApi):
+        analyzers = thehive.cortex.list_analyzers()
+        assert isinstance(analyzers, list)
+        for analyzer in analyzers:
+            assert "name" in analyzer.keys()
+            assert "id" in analyzer.keys()
+
+    def test_list_analyzers_by_type(self, thehive: TheHiveApi):
+        data_type = "mail"
+        analyzers = thehive.cortex.list_analyzers_by_type(data_type=data_type)
+        assert isinstance(analyzers, list)
+        for analyzer in analyzers:
+            assert "name" in analyzer.keys()
+            assert "id" in analyzer.keys()
```

Comment:
Same goes for this one:
```suggestion
        assert analyzers == []
```